deployments/helm-chart/README.md

```diff
@@ -79,7 +79,8 @@ Parameter | Description | Default
 `controller.service.annotations` | The annotations of the Ingress controller service. | { }
 `controller.service.loadBalancerIP` | The static IP address for the load balancer. Requires `controller.service.type` set to `LoadBalancer`.  | None
 `controller.service.externalIPs` | The list of external IPs for the Ingress controller service. | []
-`controller.serviceAccountName` | The serviceAccountName of the Ingress controller pods. Used for RBAC. | nginx-ingress
+`controller.serviceAccount.name` | The serviceAccountName of the Ingress controller pods. Used for RBAC. | nginx-ingress
+`controller.serviceAccount.imagePullSecrets.name` | The name of the secret containing docker registry credentials. | None
```

Comment:
this should be a list. See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#serviceaccount-v1-core